src/bundles/home-page-template.less

```diff
@@ -1,6 +1,49 @@
+.HomePageTemplate {
+  display: flex;
+  flex-direction: column;
+  min-width: @edu-min-page-width;
```

Comment:
Sidenote: we should increase this value in educandu, 200px is not enough in general